sysdig/internal/client/v2/model.go

```diff
@@ -605,6 +605,10 @@ type CloudauthAccountSecure struct {
 	cloudauth.CloudAccount
 }
 
+type CloudauthAccountComponentSecure struct {
+	cloudauth.AccountComponent
```

Comment:
Is there a purpose/benefit for the struct embedding here versus type alias versus neither?

Reply:
Its mainly to
- keep the convention standards, and
- if in future we want to add and process any additional fields in the provider (not backend), the struct gives us the room to do so without any major changes.

Reply:
I'll interpret this as _not really_, but I was just curious. I concede there is a place for convention following even if it yields no practical benefit, however there's also quite a bit of embedding scattered throughout this code base, which can have some negative side effects in large code bases if left unchecked. Consider this me "checking" on it. 😄